cmake/common/targets/clang-tidy.cmake

```diff
@@ -1,130 +1,18 @@
-function(sourcemeta_target_clang_tidy_attempt_install)
```

Comment:
I think I mentioned this before, but why cannot we pull ClangTidy pre-compiled binaries like before? Otherwise we are back at having it at the system level?